config/jobs/kubernetes/node-problem-detector/node-problem-detector-presubmits.yaml

```diff
@@ -79,7 +79,7 @@ presubmits:
         - bash
         - -c
         - >-
-          ./test/build.sh -p $(PULL_NUMBER) pr &&
+          ./test/build.sh -p ${PULL_NUMBER:-batch} pr &&
```

Comment:
I think this is only safe if we make the (bad) assumption that there's only one batch at a time ever.
https://github.com/kubernetes/node-problem-detector/blob/35bfe697a5fd9b058f86a3fb704b7e6d7b68457c/test/build.sh#L130

Maybe e.g. BUILD_ID

Reply:
```suggestion
          ./test/build.sh -p ${BUILD_ID} pr &&
```

Reply:
Done

Reply:
Thanks for the suggestion!